modules/async-icq/types/codec.go

```diff
@@ -50,23 +44,6 @@ func DeserializeCosmosResponse(bz []byte) (resps []abcitypes.ResponseQuery, err
 	return r.Responses, err
 }
 
-func init() {
-	RegisterLegacyAminoCodec(amino)
-	cryptocodec.RegisterCrypto(amino)
-	sdk.RegisterLegacyAminoCodec(amino)
-
-	// Register all Amino interfaces and concrete types on the authz Amino codec
-	// so that this can later be used to properly serialize MsgGrant and MsgExec
-	// instances.
-	RegisterLegacyAminoCodec(authzcodec.Amino)
-}
-
-// RegisterLegacyAminoCodec registers concrete types on the LegacyAmino codec
-func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterConcrete(Params{}, "icq/Params", nil)
-	legacy.RegisterAminoMsg(cdc, &MsgUpdateParams{}, "icq/MsgUpdateParams")
-}
-
```

Comment:
It seems the SDK still registers `RegisterLegacyAminoCodec` for modules in v0.50.1. Was there another change I am missing that does not require this anymore?

```go
// RegisterLegacyAminoCodec registers the module's types on the LegacyAmino codec.
func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
	types.RegisterLegacyAminoCodec(cdc)
}
```

Reply:
This [section](https://docs.cosmos.network/main/build/migrations/upgrading#modules-1) of the upgrade guide had mentioned that it was deprecated and no longer used. If I misread that or we think it's going to be used then we can revert this.